### DISCUSSION TITLE
Practicas inicio

### DISCUSSION THREAD

**Discussion by Shellcode-KR** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#206] [Show and tell]
A pesar de que llevo mas de la mitad de la carrera en sistemas las practicas de inicio se me dificultaron un poco, aprendi muchas cosas sobre la toma de requisitos que no habia visto antes y a usar un bonitasoft para hacer mi modelo BMP estubo interesante lo del buyer persona y el tarjet audience pero no tenia ideas sobre hacer los wireframes se me complico un poco porque no tenia muchas ideas del diseño y a pesar que busque informacion y lo hice segun mi boceto en libreta comparandolo con el trabajo que vi en las historias de insta me quedo simple y me atrase mucho, doy gracias por el tiempo extra, si a mi que tengo algunos conocimientos en esto los que empezaron desde 0 a lo mejor lo sufrieron mas

les comparto mi logo que bocete, gracias por leerme
![Abogabot](https://user-images.githubusercontent.com/43635726/156483086-5875dbf9-9f61-4a70-aa61-52184c45895a.png)

**Comment by JuanSalvador73** [comment]
Como diseñador ya he tenido experiencia en desarrollo de wireframes y UI, en aspectos de programación mas avanzada si las he padecido. Lo mejor siempre es ir al ritmo de uno mismo ya que todos tenemos conocimientos en ciertas áreas y con práctica podemos fortalecer en las que tenemos mas dificultad.

Todos de alguna manera estamos empezando de "0", y todos también queremos aprender cosas nuevas independiente de donde vengamos o que hemos estudiado.

Así que ánimo y adelante. Saludos :D
